src/MFModel.cpp

```diff
@@ -353,63 +345,56 @@ std::pair<double, double> MFModel::calc_loss(SparseDataset& dataset, uint32_t st
   double error = 0;
   uint64_t count = 0;
 
-//#ifdef DEBUG
-//  std::cout
-//    << "calc_loss() starting"
-//    << std::endl;
-//#endif
-
   for (uint64_t userId = 0; userId < dataset.data_.size(); ++userId) {
     uint64_t off_userId = userId + start_index;
-//#ifdef DEBUG
-//      std::cout
-//        << "off_userId: " << off_userId
-//        << " userId: " << userId
-//        << " dataset.data_.size(): " << dataset.data_.size()
-//        << std::endl;
-//#endif
+#ifdef DEBUG
```

Comment:
I think maybe we should remove `#define DEBUG` at the beginning of this file if we keep these debug print statements. IIRC Travis fails when the output log gets too verbose - I think that's why I commented these out initially. What do you think?